pandas/core/generic.py

```diff
@@ -3655,6 +3655,7 @@ class  max_speed
 
         self._consolidate_inplace()
 
+        indices = extract_array(indices, extract_numpy=True)
```

Comment:
Why is this needed here? The manager method has an `asarray(..)` call anyway (to convert other array-likes such as a list, or EAs, etc, so that will also convert any Index or Series passed through)

Reply:
leftover from an attempt at stronger annotating.  theres at least one test where an Index makes it through to the Block method.  will remove.